src/azure-cli/azure/cli/command_modules/security/_params.py

```diff
@@ -73,6 +74,15 @@ def load_arguments(self, _):
                 validator=validate_auto_provisioning_toggle,
                 arg_type=auto_provisioning_auto_provision_arg_type)
 
+    for scope in ['atp show']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('storage_account_name', help='Name of an existing storage account to link to.')
```

Comment:
Consider replacing "to link to" with some other phrasing, seems like there may be something more suited.

Reply:
I removed this phrase